contracts/utils/interfaces/IOrderPayable.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IOrderPayable {
+    event PaidForOrder(
+        bytes32 indexed orderId,
+        uint256 expirationTimestamp,
+        address orderSigner,
+        uint256 amount,
+        address sender
```

Comment:
I wonder if this should also be `indexed`? 🤔

Reply:
At least the intended user flow doesn't require it. We need to fetch all events and go through all `sender`s one by one to check them. Arguably even `orderId` doesn't need to be indexed.

Reply:
I agree with it being indexed, could help with things like third party services that track payments/proxy deployments etc